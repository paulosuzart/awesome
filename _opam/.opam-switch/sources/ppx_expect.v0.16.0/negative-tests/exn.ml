open! Core

let%expect_test _ =
  [%expect {| hi ho |}];
  Printexc.record_backtrace false;
  ignore (failwith "hi ho" : unit);
  [%expect {| it's off to work we go |}]
;;
