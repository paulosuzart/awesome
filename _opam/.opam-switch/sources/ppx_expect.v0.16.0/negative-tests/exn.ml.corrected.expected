open! Core

let%expect_test _ =
  [%expect {| |}];
  Printexc.record_backtrace false;
  ignore (failwith "hi ho" : unit);
  [%expect.unreachable]
[@@expect.uncaught_exn {| (Failure "hi ho") |}]
;;
